ProjectBank/Server/Model/SeedExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using ProjectBank.Infrastructure;
+
+namespace ProjectBank.Server.Model;
+
+public static class SeedExtensions
```

Comment:
Der skal mere data i denne, det gøres dog i #99 som er taget ud af committed tasks for denne uge, da andre opgaver var vigtigere.